waisntechtools/alerts/Twilio/fetch_sms.py

```diff
@@ -0,0 +1,29 @@
+from twilio.rest import Client
+from django.conf import settings
+from django.http import HttpResponseForbidden
+from twilio.request_validator import RequestValidator
+
+import requests
+import os
+
+
+def fetch_twilio(request):
+    account_sid = os.environ.get('TWILIO_ACCOUNT_SID')
+    auth_token = os.environ.get('TWILIO_AUTH_TOKEN')
+
+    validator = RequestValidator(auth_token)
+    request_valid = validator.validate(
+        request.build_absolute_uri(),
+        request.POST,
+        request.META.get('HTTP_X_TWILIO_SIGNATURE', ''))
+
+    if request_valid or settings.DEBUG:
+        client = Client(account_sid, auth_token)
+        message_fetch = client.messages.list(
+            to='+13174899826')
```

Comment:
I think this should be configurable via the UI or as an environment variable. I think the latter makes more sense to me. Thoughts?

Reply:
This was most definitely a mistake. I meant to make this an environment variable but forgot to change it